Dockerfile

```diff
@@ -1,4 +1,16 @@
-FROM registry.access.redhat.com/ubi8/nodejs-20
+FROM registry.access.redhat.com/ubi8/nodejs-20 AS test
+
+USER root
+ENV CI=1
+
+RUN npm install --location=global yarn
+
+ADD . /opt/visual-qontract
+WORKDIR /opt/visual-qontract
+RUN yarn install && yarn run lint && yarn test 
+
+
+FROM registry.access.redhat.com/ubi8/nodejs-20 AS prod
```

Comment:
can we model this like qontract server [one](https://github.com/app-sre/qontract-server/blob/master/Dockerfile) to get the benefit of multi stage? The main difference this repo just need to build static files, no need to have nodejs in the final image.

* `base`: install yarn and copy package.json & yarn.lock
* `dev`: copy files, install dependencies and build
* `test`: run lint and test
* `prod` (final): use a ubi nginx as base image, copy build result from `dev`, add entrypoint file

This can be done in a seperate card, good to get it right with a proper multistage build for a frontend repo.

Reply:
I will create a separate card for this - for now I just would like to get konflux in here - I will not have time to fix every repo.

Reply:
https://issues.redhat.com/browse/APPSRE-10978